Tools/autotest/locations.txt

```diff
@@ -58,3 +58,4 @@ SpringValley= -35.280252,149.005821,597.3,5
 Pyramid=29.9764,31.1339,0,0
 AAMEastField=39.842288,-105.212928,1809,106
 HachinoheMine=40.4539496,141.5419051,56,270
+ObamaOval=2.909368,101.655304,0,0
```

Comment:
You really, really, want to add a correct height.  Unless 0m *is* the correct height.

Very bad stuff can happen if the height is wrong.

You can get the relevant height (roughly) by mouse-overing the vehicle in SITL.  It gives you the terrain altitude.